src/Microphysics1M.jl

```diff
@@ -277,12 +306,24 @@ end
 Returns the q_rai tendency due to collisions between cloud droplets
 (autoconversion), parametrized following Kessler (1995).
 """
-function conv_q_liq_to_q_rai(param_set::APS, q_liq::FT) where {FT <: Real}
+function conv_q_liq_to_q_rai(
+    param_set::APS,
+    q_liq::FT;
+    smooth_transition::Bool = false,
+) where {FT <: Real}
 
     _τ_acnv_rai::FT = CMP.τ_acnv_rai(param_set)
     _q_liq_threshold::FT = CMP.q_liq_threshold(param_set)
 
-    return max(0, q_liq - _q_liq_threshold) / _τ_acnv_rai
+    if smooth_transition
+        k = CMP.k_thrshld_stpnss(param_set)
+        _output =
+            logistic_function_integral(q_liq, _q_liq_threshold, k) / _τ_acnv_rai
+    else
+        _output = max(0, q_liq - _q_liq_threshold) / _τ_acnv_rai
```

Comment:
```suggestion
        _output::FT = max(0, q_liq - _q_liq_threshold) / _τ_acnv_rai
```

Reply:
declaring type of _output twice is apparently not allowed although the line will be hit once!